src/views/preview/preview.scss

```diff
@@ -539,8 +359,3 @@ $medium-and-small: "screen and (max-width : #{$tablet}-1)";
         }
     }
 }
-
-.report-text textarea {
```

Comment:
This will look fine if #2108 goes in first @rschamp

Reply:
Sure — please update the description to say that this PR depends on #2108.

Reply:
done!